tests/config/nextflow.config

```diff
@@ -28,6 +30,9 @@ if ("$PROFILE" == "singularity") {
     docker.userEmulation = false
     docker.fixOwnership = true
     docker.runOptions = '--platform=linux/amd64 -u $(id -u):$(id -g)'
+    wave.enabled = true
+    wave.strategy = ['dockerfile', 'conda', 'container']
+    wave.build.repository = 'docker.io/emiller88/modules'
```

Comment:
This is not strictly necessary. if omitted it used Wave default registry (tho images will only be cached for one week)

Reply:
Ideally we'd want to not rebuild them unless the environment changes.

The other issue is it would make more sense to have one container repo per module. 

Currently it looks like wave creates a new tag with the env name rather than a image per env.